Doc/whatsnew/3.14.rst

```diff
@@ -92,6 +92,11 @@ ast
 Added :func:`ast.compare` for comparing two ASTs.
 (Contributed by Batuhan Taskaya and Jeremy Hylton in :issue:`15987`.)
 
+string
+------
+* Add :const:`visible` to :mod:`string`, containing all non-whitespace
+  printable ASCII characters.
+  (Contributed by James Murray in :gh:`119080`.)
```

Comment:
We link issues, not PRs so it should be 119067 instead of 119080 (again, sorry for the previously ill-formed suggestion).